tests/test_embed/test_interpreter.cpp

```diff
@@ -2,6 +2,9 @@
 #include <catch.hpp>
 
 #include <thread>
+#include <fstream>
+#include <cstdio>
```

Comment:
Why `< cstdio>`?

Reply:
I thought I needed it for `std::remove()` but it seems to build fine without it.